index.js

```diff
@@ -1,7 +1,50 @@
-const getRandomNumber = (min = 0, max = 100) => {
-  const randomNumber = Math.floor(Math.random() * (max - min) + min);
+const os = require('os');
+const fs = require('fs');
+const { exec } = require('child_process');
 
-  console.log(randomNumber);
+const WINDOWS_COMMAND = `powershell "Get-Process | Sort-Object CPU -Descending | Select-Object -Property Name, CPU, WorkingSet -First 1 | ForEach-Object { $_.Name + ' ' + $_.CPU + ' ' + $_.WorkingSet }"`;
+const OTHER_PLATFORM = 'ps -A -o %cpu,%mem,comm | sort -nr | head -n 1';
+
+const WINDOWS_PLATFORM = 'win32';
+let log = '';
+
+const getUnixTime = () => Math.round(new Date().getTime() / 1000);
+
+const startExecProcess = (command) => {
+  exec(command, (error, stdout, stderr) => {
+    console.clear();
+    if (stdout) {
+      console.log(`stderr: ${stdout}`);
+      log += `${getUnixTime()} : ${stdout}`;
+    }
+
+    if (stderr) {
+      console.log(`stderr: ${stderr}`);
```

Comment:
I'd recommend using `process.stdout.write` for logs instead of console (but the `console.log` uses it under the hood). Check for the difference between them